lib/libmbedtls/core/rsa.c

```diff
@@ -190,6 +190,19 @@ void crypto_acipher_free_rsa_public_key(struct rsa_public_key *s)
 	crypto_bignum_free(s->e);
 }
 
+void crypto_acipher_free_rsa_keypair(struct rsa_keypair *s)
+{
+	if (!s)
+		return;
+	crypto_bignum_free(s->e);
+	crypto_bignum_free(s->d);
+	crypto_bignum_free(s->n);
+	crypto_bignum_free(s->p);
+	crypto_bignum_free(s->q);
+	crypto_bignum_free(s->qp);
+	crypto_bignum_free(s->dp);
```

Comment:
missing `s->dq`.

Reply:
:-/ Good catch!

This needs to be fixed in master first. In 3 files it seems: `core/lib/libtomcrypt/rsa.c`, `lib/libmbedtls/core/rsa.c`, `core/drivers/crypto/caam/acipher/caam_rsa.c`...

Reply:
https://github.com/OP-TEE/optee_os/pull/4207